scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -0,0 +1,26 @@
+// CNF-1500 assembly
+[id="ztp-zero-touch-provisioning"]
+= Zero touch provisioning
+include::modules/common-attributes.adoc[]
+:context: zero-touch-provisioning
+
+Telco Edge Computing presents real-life challenges with managing hundreds to tens of thousands of clusters in hundreds of thousands locations. These challenges require fully automated management solutions with (as close as possible to) zero human interaction.
```

Comment:
s/Telco Edge Computing/Telco edge computing
"Telco" is standard to the audience.
s/real-life/extraordinary - because it really is, considering the scope of implementing 5G.
changed to fully-automated.
replaced parans with commas and reworded - "require fully-automated management solutions with, as 
closely as possible, zero human interaction.